Update economic-productivity.md (#157)
OlympusDAO-Education/Documentation: whitepapers/economic-productivity.md

**Bond control variable**: this measure is partially controlled by the policy team in order to incentivize the precise kind of treasury composition that the DAO wants. There are considerations of what kind of reserve assets the DAO wants to back the value of $OHM, such as: liquidity provider assets versus stablecoin assets versus layer 1 assets versus other tokens. Each kind of asset has different properties as reserve backing, and these must be weighted in aggregate for healthy growth and adequately stable reserve backing. The bond control variable is the internal measure of external price coordination because it sets the discount rate for buying directly from the protocol, rather than buying from a third party market maker. Thus, the bond control variable is the way that the DAO internally measures and regulates the $OHM price value in virtue of its optimal treasury composition.

> **Premium over RFV**: this is not a *policy* lever, but a market measure. It is the value that each $OHM token trades for above the amount of stablecoin value backing each token. This is a multiple, comparable to a price to earnings multiple that is familiar to value investors. The premium is the external / price measure of internal coordination; the reason that $OHM trades for a price over the RFV is because there is an external market perception of efficient internal coordination of OlympusDAO contributors. This external perception reflects investor confidence that the percentage of $OHM staked will remain high, that contributors will continue to work for the DAO, that the protocol will continue to expand its network to form new partnerships, and that demand for $OHM will continue to increase. Thus, the premium over RFV is thus a measure of the economic productivity of the DAO, and its expected future cashflows. This measure is set by the market, and not directly by the DAO policy team, but it can be influenced by policy levers.

Reviewer: ```suggestion
**Premium over RFV**: this is not a *policy* lever, but a market measure. It is the value that each $OHM token trades for above the amount of stablecoin value backing each token. This is a multiple, comparable to a price-to-earnings multiple that is familiar to value investors. The premium is the external / price measure of internal coordination; the reason that $OHM trades for a price over the RFV is because there is an external market perception of efficient internal coordination of OlympusDAO contributors. This external perception reflects investor confidence that the percentage of $OHM staked will remain high, that contributors will continue to work for the DAO, that the protocol will continue to expand its network to form new partnerships, and that demand for $OHM will continue to increase. The economy built around the Olympus ecosystem, or *econohmy*, as amiably dubbed by Zeus in [one of his blog posts](https://ohmzeus.medium.com/an-outlook-at-the-turn-of-2021-ec66a8f39a6e), will play a key role in this. The premium over RFV is thus a measure of the economic productivity of the DAO, and its expected future cash flows. This measure is set by the market, and not directly by the DAO policy team, but it can be influenced by policy levers.
```

Reviewer: Added hyphens to "price-to-earnings", and a reference to Zeus' recent post.

**4. How These Aspects Create an Economic Flywheel**

> This is an *idealized prototype* of the economic flywheel mechanism, meant to be pedagogical rather than precise in detail. It gives an intuition for how the protocol self-regulates and aligns incentives of the three main parties- market/bonders, stakers, DAO policy team. This model shows how the implementation generalizes the economic forces of supply and demand, in order to work with or counteract reflexivity in the market.

Reviewer: ```suggestion
This is an *idealized prototype* of the economic flywheel mechanism, meant to be pedagogical rather than precise in detail. It gives an intuition for how the protocol self-regulates and aligns incentives of the three main parties- market/bonders, stakers, and the DAO policy team. This model shows how the implementation generalizes the economic forces of supply and demand, in order to work with or counteract reflexivity in the market.
```

> The reward rate combined with amount of bond sales sets the rate of supply inflation,

Reviewer: ```suggestion
The reward rate combined with the amount of bond sales sets the rate of supply inflation:
```

> - higher supply → price decrease 
> - price decrease → lower premium over RFV

Reviewer: ```suggestion
- higher supply → more selling to realize profit
- more selling → price decrease
```

> - lower premium over RFV → price increase (as price reverts to standard multiple of RFV)

Reviewer: ```suggestion
- price decrease → lower premium over RFV
- lower premium over RFV → lower bond revenue
```

> - price increase → more unstaking / selling
> - more unstaking / selling → higher APY 
> - higher APY → more demand / staking (3, 3) 

Reviewer: ```suggestion
- lower bond revenue → less OHM minted
- less OHM minted → lower supply growth
- lower supply growth → more staking to capture the supply growth
- more staking → less selling pressure
- less selling pressure → price increase
- price increase → higher premium over RFV
- higher premium over RFV → more bond revenue
- more bond revenue → more OHM minted
- more OHM minted → higher supply
```
